xla/service/spmd/custom_call_handler.cc

```diff
@@ -388,6 +389,49 @@ std::unique_ptr<HloInstruction> CreateCustomCallSPMDInternal_RotateRight(
                                           kSPMDOpRotateRight, opaque);
 }
 
+absl::Status SpmdPartitioningVisitor::HandleCustomCallJaxLog(
```

Comment:
https://cs.opensource.google/tensorflow/tensorflow/+/master:third_party/xla/xla/service/custom_call_sharding_helper.h;l=77-79;drc=fa6d2e3b3053a04fa7cb940f54e30e6b25abe2a5

Maybe we can do this instead of bloating the SpmdPartitioningVisitor?